testing/integration-tests/src/full_client/client/mod.rs

```diff
@@ -412,7 +412,7 @@ async fn partial_fee_estimate_correct() {
     assert_eq!(partial_fee_1, partial_fee_2);
 }
 
-#[subxt_test]
+#[subxt_test(timeout = 120)]
```

Comment:
starting/restarting the substrate node can be quite slow sometimes, I would rather go with five minutes (300 secs)